src/Home/Home.js

```diff
@@ -1,15 +1,27 @@
 import './Home.scss'
 import { Link } from 'react-router-dom'
-import lady from './lady.png'
+import { backgroundImages } from './images/imageData'
 
 const Home = () => {
+  const generateRandomImage = () => {
+    let randomIndex = Math.floor(Math.random() * backgroundImages.length)
+    return backgroundImages[randomIndex]
+  }
+
+  const backgroundStyle = {
```

Comment:
Do you want to put this in a separate file in that directory and import it like we do modal styles? I think it's just a little cleaner. @alyssabull